datafusion/sql/src/unparser/expr.rs

```diff
@@ -76,11 +76,6 @@ pub fn expr_to_sql(expr: &Expr) -> Result<ast::Expr> {
     unparser.expr_to_sql(expr)
 }
 
-pub fn sort_to_sql(sort: &Sort) -> Result<ast::OrderByExpr> {
```

Comment:
Removed in favor of unparser sort_to_sql which respects `supports_nulls_first_in_sort` dialect setting